daemon/containerd/image_pull.go

```diff
@@ -40,6 +40,27 @@ func (i *ImageService) PullImage(ctx context.Context, baseRef reference.Named, p
 	}()
 	out := streamformatter.NewJSONProgressOutput(outStream, false)
 
+	// How long downloaded content will be held after cancellation
+	expireAt := time.Now().Add(8 * time.Hour)
+	ctx, done, err := i.client.WithLease(ctx,
+		leases.WithRandomID(),
+		leases.WithLabels(map[string]string{
+			"moby/action.pull":        baseRef.String(),
```

Comment:
Should we have a `const` for this one? (currently only used in two places, but not sure if we (should) / will be using the same label in other locations? Do we know if BuildKit also sets such labels, and if these are things we should align?